src/Toolkit/V.php

```diff
@@ -422,6 +422,25 @@ public static function __callStatic(string $method, array $arguments): bool
                V::min($value, 2) === true;
     },
 
+    /**
+     * Checks for a valid hex color
+     */
+    'hexColor' => function ($value, ?bool $hash = true): bool {
+        if (!is_string($value) || $value === '') {
+            return false;
+        }
```

Comment:
I'm asking stupid question (not big deal) because I don't know very well. Does it make sense to send the hex code as an `integer`? Like `000` or `22446688` as integer, not string. For example: `22446688` is valid hex code and `ctype_xdigit(22446688)` returns `true`.